docs/build-release/actions/scripts/git-commands.md

```diff
@@ -69,7 +69,11 @@ On the [variables tab](../../concepts/definitions/build/variables.md) set this v
 |---|---|
 | ```system.prefergit``` | ```true``` |
 
-On the [options tab](../../concepts/definitions/build/options.md) select **Allow scripts to access OAuth token**.
+### Enable scripts to access OAuth token
+- Select "Tasks" tab in the build definition

```

Comment:
It might be nicer to add some pictures here, but will leave that up to maintainers.